Simple healthchecks.io integration

ilium007 opened the pull request:
Calls helathchecks.io endpoint on success / fail. I use this for job alerting / failure to run alerting.

ilium007 committed:
Add healthchecks.io URLs to config_sample.ps1

Added healthchecks.io success and fail URLs.

ilium007 committed:
Implement health check calls in backup script

Added health check calls for success and failure scenarios.

ilium007 committed:
Add healthcheck variables to config_sample.ps1

ilium007 committed:
Disable healthcheck sending in config_sample.ps1

ilium007 committed:
Fix healthcheck endpoint calls in backup.ps1

ilium007 committed:
Update healthcheck variables in config_sample.ps1

ilium007 committed:
Fix variable casing for healthcheck parameters

ilium007 committed:
Update Restic download links to version 0.18.1

kmwoley commented:
Thanks for the PR; could you please remove the specific references to healthcheck.io in the code/comments? I think this is a fine general feature, but would like not to point to a specific service. Thanks!

ilium007 commented:
> Thanks for the PR; could you please remove the specific references to healthcheck.io in the code/comments? I think this is a fine general feature, but would like not to point to a specific service. Thanks!

Will do!

ilium007 committed:
Update healthcheck configuration to hook URLs

ilium007 committed:
Implement webhook calls for backup status

Added Invoke-HookCall function to handle webhook calls based on backup success or failure.

ilium007 committed:
Add hook URL call on backup success/failure

Added functionality to call a hook URL on success or failure of the backup process.

ilium007 commented:
Updated with generic wording. Moved URL call to end of main function so that any error is captured and fail / success URL's are called. Updated Readme.md and bumped Restic version in install.ps1

ilium007 commented:
Tested and working on Windows Server
